moped-editor/src/views/projects/projectView/ProjectSubstantialCompletionDate.js

```diff
@@ -0,0 +1,48 @@
+import { Box, Typography, Tooltip } from "@mui/material";
+import makeStyles from "@mui/styles/makeStyles";
+import { formatDateType } from "src/utils/dateAndTime";
+import { substantialCompletionDateTooltipText } from "src/constants/projects";
+
+const useStyles = makeStyles((theme) => ({
+  fieldLabel: {
+    width: "100%",
+    color: theme.palette.text.secondary,
+    fontSize: ".8rem",
+  },
+  fieldBox: {
+    maxWidth: "10rem",
+  },
+  fieldLabelText: {
+    width: "calc(100% - 2rem)",
+  },
+}));
+
+const ProjectSubstantialCompletionDate = ({ completionDate }) => {
+  const classes = useStyles();
+  return (
+    <>
+      <Typography className={classes.fieldLabel}>
+        Substantial completion date
+      </Typography>
+      <Box
+        display="flex"
+        justifyContent="flex-start"
+        className={classes.fieldBox}
+      >
+        <Tooltip
+          placement="bottom-start"
+          title={substantialCompletionDateTooltipText}
+        >
+          <Typography className={classes.fieldLabelText} component="span">
+            {
+              // If there is no input, render a "-"
+              completionDate ? formatDateType(completionDate) : "-"
+            }
```

Comment:
When the date is blank, the field label  looks a little lonely and it's slightly unclear what its purpose is.

Here's a look at rendering `Not complete` instead of `-`:
<img width="298" alt="hey" src="https://github.com/user-attachments/assets/84e60a86-b41e-4b9b-a67d-7bd57340aa77">


Also, i believe i was the person who suggested nixing some vertical space between the card title and the field label—and i don't know if i like. more space might be better?

Reply:
I thought the dash looked okay but 'Not complete' is also nice!

Reply:
"Not complete" makes me think the project isn't complete, instead of "there isnt a confirmed substantial completion date set" 

since a project can have a confirmed completion date set into the future and the project isnt completed yet. 

Maybe instead "No date set". We have the `-` when theres no ecapris number associated with a project, so users are used to loneliness.

Reply:
Y'all - let's stick with `-` for now since it is a common indicator of "no value" in the app. If the tooltip doesn't clear up any confusion, then I'm up for changing.

About the spacing, this made me realize that, in prod, we have extra margin above the headings in several tables. In this PR, @chiaberry could you update so that both the funding and phases table headings match up with the tables below them?

<img width="555" alt="Screenshot 2024-10-17 at 4 07 43 PM" src="https://github.com/user-attachments/assets/a5bf15fa-48d3-4da9-8320-3eac965d6bd1">

I'm curious if adding space between the headings and the fields (substantial completion date and eCapris ID) would give them more space to breathe.